scipy/stats/tests/test_morestats.py

```diff
@@ -2460,106 +2463,81 @@ class TestCircFuncs:
                              [(stats.circmean, 0.167690146),
                               (stats.circvar, 0.006455174270186603),
                               (stats.circstd, 6.520702116)])
-    def test_circfuncs(self, test_func, expected):
-        x = np.array([355, 5, 2, 359, 10, 350])
-        assert_allclose(test_func(x, high=360), expected, rtol=1e-7)
-
-    def test_circfuncs_small(self):
-        x = np.array([20, 21, 22, 18, 19, 20.5, 19.2])
-        M1 = x.mean()
+    def test_circfuncs(self, test_func, expected, xp):
+        x = xp.asarray([355., 5., 2., 359., 10., 350.], dtype=xp.float64)
+        xp_assert_close(test_func(x, high=360), xp.asarray(expected, dtype=xp.float64),
+                        rtol=1e-7)
```

Comment:
With the change in default tolerance, this can be
```suggestion
        x = xp.asarray([355., 5., 2., 359., 10., 350.])
        xp_assert_close(test_func(x, high=360), xp.asarray(expected))
```